ts/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
+
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
+.idea
+.DS_Store
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
```

Comment:
Personal taste: I generally prefer to have a minimal .gitignore file containing only the files I expect to be present in the system (eg .node_modules, dist etc).

eg: given we are using pnpm, I'd rather `git status` tell me if yarn-*.log files are present, in which I case I will manually remove them.

I don't feel strongly enough to argue this.